src/virtual_dom/vcomp.rs

```diff
@@ -198,7 +199,10 @@ impl VDiff for VComp {
                     }
                     Reform::Before(next_sibling) => {
                         // Temporary node which will be replaced by a component's root node.
-                        let dummy_node = document().create_text_node("");
+                        let dummy_node = parent
+                            .owner_document()
+                            .unwrap("Parent node not attached to a document")
```

Comment:
Did you mean `.expect` here?

Reply:
Most likely yeah, I'm actually not certain on the difference in this context.

Reply:
Both panic if the unwrapped instance is `None` for `Option<T>` or  `Err` for `Result<T, E>`. The signature of [unwrap][] is `pub fn unwrap(self) -> T` and for [expect][] it's `pub fn expect(self, msg: &str) -> T`. The latter takes a custom panic message, while the former does not. So in this line you could either remove the argument to `unwrap` or switch it to `expect`.

```suggestion
                            .expect("Parent node not attached to a document")
```

[unwrap]: https://doc.rust-lang.org/std/option/enum.Option.html#method.unwrap
[expect]: https://doc.rust-lang.org/std/option/enum.Option.html#method.expect